test/quantization/quantize_/workflows/int4/test_int4_opaque_tensor.py

```diff
@@ -75,7 +77,23 @@ def test_module_path(self, dtype):
                 str(type(state_dict["weight"])),
                 "<class 'torchao.quantization.Int4OpaqueTensor'>",
             )
+    def test_activation_prescaling(self):
+        dtype = torch.bfloat16
+        input = torch.randn(1, 128, dtype=dtype)
+        linear = torch.nn.Linear(128, 256, bias=False, dtype=dtype)
+        original = linear(input)
+        quantize_(linear, get_config(group_size=128))
+        qw = linear.weight
+        assert isinstance(qw, SupportsActivationPreScaling), (
+            "Expected int4 tensor supports activation prescaling"
+        )
```

Comment:
Could you explain why `qw` is an instance of `SupportsActivationPreScaling`? `qw` should be an instance of `Int4OpaqueTensor`, right?

Reply:
Thanks for the explanation